OrbitCore/CaptureData.h

```diff
@@ -132,6 +140,19 @@ class CaptureData {
     callstack_data_->AddCallstackEvent(std::move(callstack_event));
   }
 
+  void AddUniqueTracepointEventInfo(uint64_t key,
+                                    orbit_grpc_protos::TracepointInfo tracepoint_info) {
+    tracepoint_info_manager_->AddUniqueTracepointEventInfo(key, std::move(tracepoint_info));
+  }
+
+  void AddTracepointEvent(orbit_client_protos::TracepointEventInfo tracepoint_event_info) {
+    tracepoint_info_manager_->AddTracepointEvent(std::move(tracepoint_event_info));
+  }
+
+  void AddTracepointEvent(uint64_t time, uint64_t tracepoint_hash, int32_t thread_id) {
+    tracepoint_event_buffer_->AddTracepointEvent(time, tracepoint_hash, thread_id);
+  }
+
```

Comment:
Do/will you use/need both these methods or is only one of the two used? In the latter case, I would remove the one that is not necessary, let's try to have only necessary methods in this big class.

Reply:
Yes, both of them are used are are necessary.

Reply:
One is for mapping the tracepoint event to the hash key and the second is for mapping to the thread is and timestamp.

Reply:
If the do different things (I see that one uses `tracepoint_info_manager_` and one uses `tracepoint_event_buffer_`) they should have different names. 

Besides, why do you need to add tracepoint events to both? Do they both store them? Why? Shouldn't they be stored only in `TracepointEventBuffer`?

Reply:
The events are added to TracepointInfoManager such that they could be later used for statistics. However, if you this it would be better to delete saving the tracepoint events for now, I can do it.

Reply:
Duplicating all the events for possible later usage for statistics sounds wrong, I think we should get rid of that and later have a better way to keeps statistics.

Reply:
Done